Source/Urho3D/Core/WorkQueue.h

```diff
@@ -55,12 +80,10 @@ class WorkQueueVector : public MultiVector<T>
     T& Emplace(Args&& ... args);
 };
 
-/// Task function signature.
-/// TODO: Get rid of parameter
+/// Deprecated.
 using WorkFunction = ea::function<void(unsigned threadIndex)>;
```

Comment:
Maybe we should start using `[[deprecated]]`?

https://en.cppreference.com/w/cpp/language/attributes/deprecated

Reply:
Then we will be getting annoying warnings from old renderer... Having said that, I am all for [deprecated] when engine itself doesn't use the feature